src/main/java/ch/cern/eam/wshub/core/services/workorders/SafetyService.java

```diff
@@ -0,0 +1,65 @@
+package ch.cern.eam.wshub.core.services.workorders;
+
+import ch.cern.eam.wshub.core.annotations.Operation;
+import ch.cern.eam.wshub.core.client.InforContext;
+import ch.cern.eam.wshub.core.interceptors.LogDataReferenceType;
+import ch.cern.eam.wshub.core.services.INFOR_OPERATION;
+import ch.cern.eam.wshub.core.services.entities.BatchResponse;
+import ch.cern.eam.wshub.core.services.workorders.entities.EntitySafetyWSHub;
+import ch.cern.eam.wshub.core.tools.InforException;
+import ch.cern.eam.wshub.core.tools.Tools;
+import net.datastream.schemas.mp_entities.entitysafety_001.EntitySafety;
+
+import java.util.List;
+
+
+public interface SafetyService {
+    @Operation(logOperation = INFOR_OPERATION.SAFETY_GET, logDataReference1 = LogDataReferenceType.RESULT)
+    EntitySafety getEntitySafety(InforContext context, String safetyCode) throws InforException;
+
+    @Operation(logOperation = INFOR_OPERATION.SAFETY_GET_BATCH, logDataReference1 = LogDataReferenceType.RESULT)
+    BatchResponse<EntitySafety> getEntitySafetiesBatch(InforContext context, List<String> safetyCodes) throws InforException;
+
+    @Operation(logOperation = INFOR_OPERATION.SAFETY_ADD, logDataReference1 = LogDataReferenceType.RESULT)
+    String addSafety(InforContext context, EntitySafetyWSHub safety) throws InforException;
```

Comment:
Doesn't the next method make this one obsolete?

Reply:
In general, I would argue to leave it there, as it provides a simpler interface in case the user wants to do something simple (no need to create a 1 element list if the user is sure that only 1 will be added, plus it provides a little bit of semantic safety, as it is makes sure only one will be added).

In any case, let me know if it should be deleted for another reason, and I'll do it, tho.